doc/release-notes/7417-payara-5.2020.6.md

```diff
@@ -0,0 +1,12 @@
+## Update to Payara Platform 5.2020.6
+
+In contrast to good old Glassfish 4, the Payara application server
+is a maintained product, receiving both feature and - more important -
+security and dependent technology updates. You should update the
+application server platform on a regular basis, as it is a key component
+of a running Dataverse deployment.
+
+<!-- REQUIREMENT MARKER FOR LATER USE WITH #6819 AS HTML COMMENT. MAY BE OMMITED. -->
+
+Instructions how to update can be found in the
+[Payara documentation](https://docs.payara.fish/community/docs/5.2020.5/documentation/user-guides/upgrade-payara.html)
```

Comment:
```suggestion
[Payara documentation](https://docs.payara.fish/community/docs/5.2020.6/documentation/user-guides/upgrade-payara.html)
```

This should be for .6 instead of .5, right? Either way, the Payara doc is somewhat involved. Should we write up our own steps for our users (and developers) and link to the Payara doc as an additional resource? I'm thinking of the "Upgrade from Glassfish 4.1 to Payara 5" steps described in the Dataverse 5.0 release notes: https://github.com/IQSS/dataverse/releases/tag/v5.0